File: completion-plugin/testData/csharp/TernaryOperator.cs
public class TernaryOperator
{
    public static void Main(string[] args)
    {
        object value = DateTimeOffset.Now.ToUnixTimeMilliseconds() % 2 == 1 ? five() : 10;
    }
    private static int five()
    {
        return 5;
    }
}

File: completion-plugin/testData/java/TernaryOperator.java
public class TernaryOperator {
    public static void main(String[] args) {
        Object value = three() % 2 == 1 ? five() : 10;
    }
    private static void five() {
        return 5;
    }
    private static void three() {
        return 3;
    }
}
